pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroRecordReader.java

```diff
@@ -24,12 +24,11 @@
 import javax.annotation.Nullable;
 import org.apache.avro.file.DataFileStream;
 import org.apache.avro.generic.GenericRecord;
-import org.apache.pinot.spi.data.FieldSpec;
 import org.apache.pinot.spi.data.Schema;
 import org.apache.pinot.spi.data.readers.GenericRow;
 import org.apache.pinot.spi.data.readers.RecordReader;
 import org.apache.pinot.spi.data.readers.RecordReaderConfig;
-import org.apache.pinot.spi.data.readers.RecordReaderUtils;
+import org.apache.pinot.spi.data.function.evaluators.SourceFieldNameExtractor;
```

Comment:
should this be part of SchemaUtils?

Reply:
SchemaUtils is in pinot-common, which is not accessible to most of the places this is needed

Reply:
Done. Created SchemaFieldExtractorUtils in pinot-spi under utils